packages/get-platform/src/link.ts

```diff
@@ -0,0 +1,8 @@
+import chalk from 'chalk'
+import terminalLink from 'terminal-link'
+
+export function link(url): string {
+  return terminalLink(url, url, {
+    fallback: (url) => chalk.underline(url),
+  })
+}
```

Comment:
This is currently copied from `@prisma/internals` due to circular dependencies concerns.